crypto/ledger.go

```diff
@@ -0,0 +1,18 @@
+// +build ledger
```

Comment:
Can this be `cgo,ledger` just so that this doesn't work if ledger flag is set but cgo isn't installed.

Reply:
Yes 👍